app/src/main/java/com/example/bookwormadventuresdeluxe2/Activities/AddOrEditBooksActivity.java

```diff
@@ -222,6 +213,24 @@ private void deletePhoto()
      */
     public void saveBook(View view)
     {
+        /* Disable saving changes if book is not available */
+        if (this.editingBook
+                && Arrays.asList(Status.Requested,
+                                    Status.Accepted,
+                                    Status.bPending,
+                                    Status.Borrowed,
+                                    Status.rPending).contains(bookToEdit.getStatus()))
+        {
+            Toast toast = Toast.makeText(AddOrEditBooksActivity.this, getString(R.string.cannot_edit_book), Toast.LENGTH_LONG);
```

Comment:
Can we customize the message for each case? Something like:

- you can't edit because this book is borrowed
- you can't edit because this book has been requested
- etc.

Reply:
✔ ill do it tomorrow

Reply:
Lol that makes sense... I'd be surprised if you submitted another commit this late.